Documentation/Books/Manual/Views/ArangoSearch/DetailedOverview.md

```diff
@@ -7,9 +7,9 @@ filtering on multiple document attributes.
 
 ## View datasource
 
-The IResearch functionality is exposed to ArangoDB via the the ArangoSearch view
-API because the ArangoSearch view is merely an identity transformation applied
-onto documents stored in linked collections of the same ArangoDB database.
+The IResearch functionality is exposed to ArangoDB via the view API for views
```

Comment:
Is it relevant to call it `IResearch` here?

Reply:
Yes. Properties of ArangoSearch view rely on internal data structures of IResearch.

Reply:
True, but do we need to say that? Probably more a marketing question than a technical one.